### ISSUE TITLE
review clock and file times on USB on EM and BS

### ISSUE THREAD

**Issue by benadida** [votingworks/vxsuite#115]
The modification time of files is written in UTC without timezone, so all files look like they were modified in the future.

**Comment by eventualbuddha** [comment]
So far I haven't been able to reproduce this. I set up a VM with vxsuite-complete-system as an Election Manager, locked down with `setup-machine.sh`. Both before and after I messed with the time zone of the machine the ballot packages saved had sensible timestamps when I looked at them in the picker. Both "sample" files were saved around 2:40pm-ish local time, or 14:40 as it says.

![image.png](https://zube.io/files/votingworks/d3700cccb57e5225534357d231fe8ae7-image.png)

Anyone have hints on reproducing this?

**Comment by eventualbuddha** [comment]
I continue to be unable to reproduce this reliably, even on real hardware. I've tried setting the system clock to Central Time and writing a ballot package, which wrote the file with the current time. Because FAT32 doesn't record timezones and doesn't use UTC, it stored it with "local" time. When I viewed the file on another computer in Pacific Time it showed it the file at the same "local" time as when I saved it, not accounting for timezone offset. This is expected, given FAT32's non-handling of timezones.

Maybe we should be formatting the drives as NTFS? I think everything should be able to read/write that and it uses UTC as the implicit timezone.
